packages/cli/src/utils/compiler.ts

```diff
@@ -94,33 +92,8 @@ export const compileBlock = async ({ projectPath, entryFile, outputName }: Compi
 };
 
 export const compilePlatformApp = async ({ outputName, entryFile, projectPath = '' }: CompilerOptions) => {
-    const getHash = (text) => createHash('sha256').update(text).digest('hex');
     const appBridgeVersion = getAppBridgeVersion(projectPath);
 
-    const htmlHashPlugin: PluginOption = {
```

Comment:
Lets split the compile in a follow up so it doesn't get noisy is something gets changed in here, and each team can fully own their compile 👍